lblod/ember-rdfa-editor-lblod-plugins
# Fix several type issues

### Overview
This PR is a series of modifications/updates in order to improve the types of this project:
- Removal of the now redundant types for the `string-set` ember-data transform
- Fix issues with the `tracked-toolbox` types
- Update of the `@ember/test-helpers` package to 2.9.3

##### connected issues and PRs:
None

### How to test/reproduce
- Remove `skipLibCheck: true` from tsconfig.json
- Run `ember ts:precompile`
- You'll notice that a lot of type errors (in comparison to master) have been resolved, only some type errors related to `ember-intl` remain

### Challenges/uncertainties
- Before we can remove the `skipLibCheck: true` from tsconfig.json alltogether, we need to resolve the type issues with `ember-intl`

### Checks PR readiness
